sdks/python/apache_beam/examples/inference/multi_language_inference/expansion_service/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
```

Comment:
Package "expansion_service" is too generic. Can you move to a package name that is specific to this example ?

Reply:
renamed the package to `multi_language_custom_transform`